File: JAVA/Daily Flash Codes/Week1/19August/program4.java

/*
 * Program 4: Write a Program to Print following Pattern
1 2 3 4
1 2 3 4
1 2 3 4
1 2 3 4
*/

import java.io.*;

class Program {
	public static void main(String args[]) throws IOException {
		int c;
		System.out.println("Enter num of rows: ");
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in) );
		c = Integer.parseInt(br.readLine() );		
		
		for(int olc = 0; olc < c; olc++ ) {
			for(int ilc = 1; ilc <= c; ilc++ ) {
				System.out.print(ilc +" ");
			}
			System.out.println();
		}

			
	}
}

File: C++/Daily Flash/week1/Day2_Jan14/code_c++/program4.cpp
/*
Program 4: Write a Program to Print following Pattern
1 1 1 1
1 1 1 1
1 1 1 1
1 1 1 1
*/


#include <iostream>

int main() {
	for(int olc = 0; olc<4; olc++) {
		for(int ilc = 0; ilc<4; ilc++) {
			std::cout << " 1 " ;
		}
		std::cout<<std::endl;
	}
	return 0;
}


